rskj-core/src/main/java/co/rsk/peg/Federation.java

```diff
@@ -152,33 +140,12 @@ public boolean isMember(FederationMember federationMember){
         return this.members.contains(federationMember);
     }
 
-    @Override
```

Comment:
It's ok to override this method, it's part of the `Object` java class from which all clases inherit